packages/notification-center/src/components/notification-center-component.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
this is all-in-one NC React component consisting of: `NovuProvider, PopoverNotificationCenter, NotificationBell`